### ISSUE TITLE
NestJs CLI prints strange message 

### ISSUE THREAD

**Issue by ulisses-cruz** [nestjs/nest-cli#1480]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

![image](https://user-images.githubusercontent.com/11726278/148699971-caf6f7c3-a1f1-4eee-b23b-4dcf761ff160.png)

### Minimum reproduction code

npx @nestjs/cli -h

### Steps to reproduce

npx @nestjs/cli -h

### Expected behavior

should show the help message for the nestjs cli

### Package version

last

### Node.js version

17.2.0

### In which operating systems have you tested?

- [X] macOS
- [ ] Windows
- [ ] Linux

### Other

If it is not a problem of my computer, maybe nestjs cli repo was hacked

**Comment by DylanMorison** [comment]
I am having this problem as well on ubuntu

**Comment by pamelaschuldiner** [comment]
you can read all this here -> https://github.com/Marak/colors.js/commit/074a0f8ed0c31c35d13d28632bd8a049ff136fb6

**Comment by dpnilsen** [comment]
Same issue on Windows 10. Tried to uninstall and install older version (like 8.1.4, same issue though)

**Comment by NickKelly1** [comment]
Same issue on Ubuntu 21.10

Tried on NodeJS v14.17.4 and v17.3.0

**Comment by Tony133** [comment]
This is a problem with the `color.js` package

https://github.com/Marak/colors.js/issues/285

**Comment by CodingVampyre** [comment]
Same problem for me.
- Windows 11 and MacOS
- Nestjs 8.0.0

**Comment by DylanMorison** [comment]
Is there a solution for this that anyone has found?

**Comment by pamelaschuldiner** [comment]
> Is there a solution for this that anyone has found?

Well... you can always fix the version in your package-lock file to the last stable.
This is a protest. Nest shouldn't be depending on this package anymore I think... When is Nest (and others depending on this) going to change this? don't know xD

**Comment by ElvisKnapman** [comment]
same issue all of a sudden in the last hour or so

- Windows 11
- NestJS v8.0.0
- Node.js v16.13.1

**Comment by liquidautumn** [comment]
@pamelaschuldiner Nest has colors dependency through `cli-table3` which has pinned `colors` version already and going to phase out it completely in favor of `chalk`. This package will be fixed when #1478 merged and released.

**Comment by pamelaschuldiner** [comment]
> @pamelaschuldiner Nest has colors dependency through `cli-table3` which has pinned `colors` version already and going to phase it out completely in favor of `chalk`. This package will be fixed when #1478 merged and released.

Beautiful then! Thanks!

**Comment by micalevisk** [comment]
Thus there's nothing we can do (neither on `@nestjs/cli` source) besides waiting for the next release of `@nestjs/cli` which depends on the next release of `cli-table3`.

---

[dependency graph of `@nestjs/cli`](https://npmgraph.js.org/?q=%40nestjs%2Fcli)

**Comment by liquidautumn** [comment]
@micalevisk it can be hotfixed unless real fixes go through the pipeline, ie https://github.com/aws/aws-cdk/pull/18324#issuecomment-1008262155

**Comment by Duduzera1997** [comment]
+1

NestJS CLI: Latest
NodeJS: 14.18
S.O: MacOS Big Sur

Same problem in Docker build in Github Actions.

**Comment by jmcdo29** [comment]
We're aware of the problem, and the solution. Thanks everyone for letting us know. Thank you @liquidautumn for pointing out the PR from renovate as well. We'll have this merged and patched as soon as we can. 

_Please do not keep adding "+1" or "Same problem here" messages. They add a lot of noise to the day. Simply subscribe to the issue or use reactions to show support._

**Comment by jmcdo29** [comment]
Also, until we have this PR made, if you need a way around you can use the `resolutions` property of your `package.json` to pin `colors` to `1.4.1` if you use `pnpm` or `yarn`, or you can use [`npm-force-resolutions`](https://www.npmjs.com/package/npm-force-resolutions) for resolve colors to the same version for npm

```json
...
  "resolutions": {
    "colors": "1.4.1"
  }
```

**Comment by kachkaev** [comment]
`"colors": "1.4.1"` is also affected – see [version history on npm](https://www.npmjs.com/package/colors?activeTab=versions) and [v1.4.1. on `unpkg`](https://unpkg.com/browse/colors@1.4.1/lib/index.js). The latest safe version to date is `1.4.0`.

**Comment by kyawswarthwin** [comment]
Win 11
Node v16.13.1
npm 8.3.0

any temporary solution to work "nest new project-name"?

**Comment by jmcdo29** [comment]
`git clone` the [typescript-starter repository](https://github.com/nestjs/typescript-starter) and resolve the issue in the local repo. Not much else to be done right now

**Comment by quyennt21** [comment]
@kyawswarthwin go to your_project/node_module find @nestjs/cli > add   

  "overrides": {
    "colors": "1.4.0"
  }

And run re-install package json

**Comment by kyawswarthwin** [comment]
@quyennt21 thanks it works.

**Comment by kyawswarthwin** [comment]
@quyennt21 I can't start my nest project with npm run start. It shows the same error.

**Comment by quyennt21** [comment]
> @quyennt21 I can't start my nest project with npm run start. It shows the same error.

Change package json in Your_project/node_modules/ nestjs cli. Do the same with global package.

**Comment by Mark24Code** [comment]
same issues

**Comment by heepaanee-at-cordial** [comment]
it is a temporary fix!

add your package.json
"overrides": {
  "cli-table3": "0.6.1"
}

remove package-lock.json
remove node_modules

do again "npm install"

**Comment by heepaanee** [comment]
it is a temporary fix!

add to your package.json
```
"overrides": {
  "cli-table3": "0.6.1"
}
```

remove unnecessaries
- remove node_modules
- remove package-lock.json

re-install npm modules
```
npm install
```

**Comment by kamilmysliwiec** [comment]
This should be fixed in 8.1.8! Thanks for reporting
